src/Symfony/Bundle/TwigBundle/composer.json

```
@@ -22,7 +22,7 @@
         "symfony/http-kernel": "^4.4",
         "symfony/polyfill-ctype": "~1.8",
         "symfony/polyfill-php80": "^1.16",
-        "twig/twig": "^1.43|^2.13|^3.0.4"
+        "twig/twig": "^1.43|^2.14|^3.3.8"
```

Should include the bugfix version.
```suggestion
        "twig/twig": "^1.43|^2.14.11|^3.3.8"
```

---

Done, changed in

- composer.json
- src/Symfony/Bridge/Twig/composer.json
- src/Symfony/Bundle/FrameworkBundle/composer.json
- src/Symfony/Bundle/SecurityBundle/composer.json
- src/Symfony/Bundle/TwigBundle/composer.json
- src/Symfony/Bundle/WebProfilerBundle/composer.json
- src/Symfony/Component/HttpKernel/composer.json
- src/Symfony/Component/VarDumper/composer.json